src/content/docs/en/guides/server-side-rendering.mdx

```diff
@@ -31,9 +40,80 @@ To decide which one to use in your project, choose the `output` option that repr
 
+### Add an Adapter
+
+To deploy a project in `server` or `hybrid` mode, you need to add an **adapter**. This is because both of these modes require a server _runtime_: the environment that runs code on the server to generate pages when they are requested. Each adapter allows Astro to output a script that runs your project on a specific runtime, such as Vercel, Netlify or Cloudflare.
+
+You can find both [official and community adapters in our integrations directory](https://astro.build/integrations/?search=&categories%5B%5D=adapters). Choose the one that corresponds to your [deployment environment](/en/guides/deploy/).
+
+#### `astro add` install
+
+You can add any of the [official adapters maintained by Astro](/en/guides/integrations-guide/#official-integrations) with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step. 
+
+For example, to install the Vercel adapter, run:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npx astro add vercel
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm astro add vercel
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn astro add vercel
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+#### Manual Install
+
+You can also add an adapter manually by installing the package and updating `astro.config.mjs` yourself.
+
+For example, to install the Vercel adapter manually:
+
+1. Install the adapter to your project dependencies using your preferred package manager:
+
+   <PackageManagerTabs>
+     <Fragment slot="npm">
+     ```shell
+     npm install @astrojs/vercel
+     ```
+     </Fragment>
+     <Fragment slot="pnpm">
+     ```shell
+     pnpm install @astrojs/vercel
+     ```
+     </Fragment>
+     <Fragment slot="yarn">
+     ```shell
+     yarn add @astrojs/vercel
+     ```
+     </Fragment>
+   </PackageManagerTabs>
+
+2. [Add the adapter](/en/reference/configuration-reference/#adapter) to your `astro.config.mjs` file's import and default export, along with your desired `output` mode:
+
+    ```js ins={3,7} {6}
+    // astro.config.mjs
+    import { defineConfig } from 'astro/config';
+    import vercel from '@astrojs/vercel/serverless';
+
+    export default defineConfig({
+      output: 'server',
+      adapter: vercel(),
+    });
+    ```
+
+    Note that different adapters may also have different configuration settings. Read each adapter's documentation, and apply any necessary config options to your chosen adapter in `astro.config.mjs`
+
 ### Configure `server` or `hybrid`
 
-To enable on-demand rendering, first update your `output` configuration to one of the two server-rendered modes.
+To enable on-demand rendering, you will need to update your `output` configuration to one of the two server-rendered modes.
```

Comment:
```suggestion
To enable on-demand rendering, you must update your `output` configuration to one of the two server-rendered modes.
```

Just a small nit here, instead of "do it later," "do it now!"

Probably meaningless, just my two cents! :)

Reply:
Sure, I'll take it!